src/main/java/org/junit/rules/ExpectedException.java

```diff
@@ -14,7 +14,10 @@
 
 /**
  * The {@code ExpectedException} rule allows you to verify that your code
- * throws a specific exception.
+ * throws a specific exception. Note that {@link org.junit.Assert#assertThrows Assert.assertThrows}
```

Comment:
I agree with @eamonnmcmanus that the note about `assertThrows` should be at the top, because it is an important message for all Java 8 users.